vip-helpers/vip-elasticsearch.php

```diff
@@ -62,18 +62,36 @@ function es_api_search_index( $args ) {
 
     $args['blog_id'] = absint( $args['blog_id'] );
 
-    $service_url = 'https://public-api.wordpress.com/rest/v1/sites/' . $args['blog_id'] . '/search';
+    $endpoint = sprintf( '/sites/%s/search', $args['blog_id'] );
+    $service_url = 'https://public-api.wordpress.com/rest/v1' . $endpoint;
+
+    $do_authenticated_request = false;
+    if ( class_exists( 'Jetpack_Client' )
+            && isset( $args['authenticated_request'] )
```

Comment:
You'll want to load and use the vip adapter (https://github.com/alleyinteractive/es-wp-query/blob/master/adapters/wpcom-vip.php) to make sure that the WP_Query args => ES fields are mapped correctly (and any other custom logic it includes).

You can obviously copy the adapter and modify it as needed but will need to keep it in sync with any future updates to the vip adapter, so it would be better just to use the filter to flag it as an `authenticated_request`.